msgraph/application_templates.go

```diff
@@ -94,10 +95,14 @@ func (c *ApplicationTemplatesClient) Instantiate(ctx context.Context, applicatio
 		return nil, status, fmt.Errorf("json.Marshal(): %v", err)
 	}
 
+	// This post call can often timeout and return a 404, particularly against US Gov cloud. The aim here is to give it longer to complete on azure's backend
+	// before timeout occurs
+	ctx, cancel := context.WithTimeout(ctx, 45*time.Second)
+	defer cancel()
+
```

Comment:
As discussed, I'm not sure this will have the desired effect, since the root cause is a 404 response despite the POST request being successful (somewhat egregious deviation from HTTP specs).

```suggestion
```